component/component.md

```diff
@@ -263,6 +263,50 @@ only renders friendly messages:
       }
     });
 
+### LeakScope
```

Comment:
I think leakScope should have it's own page, similar to how all the other prototype properties do.